test/fixtures/output/php/curl/multipart-file.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+$postData = "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\n\r\n-----011000010111000001101001--\r\n";
```

Comment:
Thoughts on introducing some splitting on `\n` to transform this line into something like the following?

```php
$postData = "-----011000010111000001101001\r\n";
$postData += "Content-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\n";
$postData += "Content-Type: text/plain\r\n\r\n\r\n-----011000010111000001101001--\r\n";
```

Reply:
I think some changes along those lines would be good; but should be saved for a different pull (that and I don't have a great solution in my head at this moment).